### ISSUE TITLE
** Doesn't work recursively

### ISSUE THREAD

**Issue by GuyHadas** [htmllint/htmllint-cli#15]
[app.zip](https://github.com/htmllint/htmllint-cli/files/419050/app.zip)

Getting script to find all the html files is not doable with ** in path.

**Comment by juanjoDiaz** [comment]
Any chance of fixing this soon??

Just to elaborate a bit more... Let's say we have a project as:
```
|-- src
    | -- css
        | -- ...
    | -- html
        | -- module1
            | -- ...
        | -- module2
            | -- ...
        | -- index.html
    | -- js
        | -- ...
```

If I do `htmllint src/html/**/*.html`, only `src/html/index.html` will be linted. On the other hand, If I do `htmllint src/html/**/**/*.html`, only the contents of `src/html/module1` and `src/html/module2` will be linted

**Comment by Potherca** [comment]
Until this is resolved, a variation of the following might work for you:

    htmllint $(find . -name '*.html')

**Comment by mlochbaum** [comment]
FINALLY had a look at this. It seems like the only problem is that the shell is catching these patterns and expanding them incorrectly before they get to htmllint. To avoid this enclose in double quotes: `htmllint "src/html/**/*.html"`. This is awkward, but other commands like find have similar problems so it's nothing unheard of.

Can someone confirm that enclosing in quotes works correctly?

**Comment by compeek** [comment]
> FINALLY had a look at this. It seems like the only problem is that the shell is catching these patterns and expanding them incorrectly before they get to htmllint. To avoid this enclose in double quotes: `htmllint "src/html/**/*.html"`. This is awkward, but other commands like find have similar problems so it's nothing unheard of.
> 
> Can someone confirm that enclosing in quotes works correctly?

@mlochbaum I realize this is very old, but thank you for the suggestion. It works for me.
